ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -19,6 +19,9 @@
   # Roles #
   #########
   roles:
+    - role: logs
+      position: "Start"
```

Comment:
interesting! i did not know this "position" in ansible

Reply:
Oh thats just how I decided to name the variable, I don't think its an ansible standard variable (if it is then it is purely a coincidence haha)